python_modules/libraries/dagster-snowflake-pyspark/dagster_snowflake_pyspark/snowflake_pyspark_type_handler.py

```diff
@@ -0,0 +1,163 @@
+from typing import Mapping
+
+import dagster._check as check
+from dagster import InputContext, MetadataValue, OutputContext, TableColumn, TableSchema
+from dagster._core.definitions.metadata import RawMetadataValue
+from dagster._core.storage.db_io_manager import DbTypeHandler, TableSlice
+from dagster_snowflake import build_snowflake_io_manager
+from pyspark.sql import DataFrame, SparkSession
+
+SNOWFLAKE_CONNECTOR = "net.snowflake.spark.snowflake"
+
+
+def _get_sf_options(config, table_slice):
```

Comment:
Nitpick: add type annotations. Also, I think `get_snowflake_options` would be a clearer name (I know it's probably my fault for originating the abbreviation.